### ISSUE TITLE
Mandala component is exposed to the Accessibility tree and is read out loud by VoiceOver (macOS)

### ISSUE THREAD

**Issue by erickwilder** [mdn/yari#6332]
### Summary

The `<Mandala />` component, used in the Homepage is exposed in the Accessibility tree which makes is passive to be read out loud by screen readers like VoiceOver on macOS. 

### URL

https://developer.mozilla.org/en-US/

### Reproduction steps

1. Open MDN Web Docs with VoiceOver enabled
2. Navigate until the end of the main Hero section search component
3. Use VO-Right Arrow key to navigate once more (the Mandala component is focused by VoiceOver)
4. Tap the right arrow key and the whole contents of the mandala component will be read out loud together (i.e slash-slash-slash-slash...)

Alternatively, reading the page from top-to-bottom can also trigger the behaviour, but it takes more time and effort to reproduce

### Expected behavior

Only the relevant content on the page should be read out loud

### Actual behavior

All loose characters that composes the Mandala are read out loud (i.e slash, slash, slash, slash, and so on)

### Device

Desktop

### Browser

Safari

### Browser version

Stable

### Operating system

Mac OS

### Screenshot

<img width="499" alt="Screenshot 2022-05-21 at 12 24 39" src="https://user-images.githubusercontent.com/607262/169649042-824082e8-465c-4c3d-ab1c-23edcf14aca1.png" alt="Screenshot of Firefox Accessibility panel in its Developer Tools highlighting how the Mandala component is exposed in the Accessibility tree">

A screen recording is available as an unlisted video on YouTube: https://youtu.be/ILD7FyeseRU

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).
